resources/create-sa.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+
+if [ -z "$1" ]; then
+    echo "usage create-sa.sh servicename [ro|rw]"
+    exit 1
+fi
+
+SERVICE="$1"
+
+if [ -z "$2" ] || [[ "$2" == "ro" ]]; then
+    PERMISSION="repository:read,pull:*"
+elif [[ "$2" == "rw" ]]; then
+    PERMISSION="*"
```

Comment:
I think we should use `repository:*` for `rw`. We could add `admin` for `*` as third option.